Mage/src/main/java/mage/game/jumpstart/JumpstartPoolGenerator.java

```diff
@@ -0,0 +1,82 @@
+package mage.game.jumpstart;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.CharSource;
+import com.google.common.io.Resources;
+
+import mage.cards.Card;
+import mage.cards.decks.Deck;
+import mage.cards.decks.DeckCardInfo;
+import mage.cards.decks.DeckCardLists;
+import mage.game.GameException;
+
+public class JumpstartPoolGenerator {
```

Comment:
Miss comments about format and data source (like wiki page with description, source for packs list, etc)

Reply:
Added a description and links about the format.  I'll update the file with the details of the packs once they're released.